<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlowerViewClass</class>
 <widget class="QWidget" name="FlowerViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlowerView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#FlowerViewClass
{
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(44, 40, 40, 255), stop:1 rgba(128, 128, 128, 255));
	border-radius:4px;
	padding: 0px;
	margin: 0px;
}

QWidget#FlowerViewClass&gt;QFrame#frame
{
    background:red;
	padding: 1px;
	margin: 1px;
}

QWidget#FlowerViewClass&gt;QPushButton
{
	padding: 0px;
	margin: 0px;
	border:none
}

QWidget#FlowerViewClass&gt;QToolButton
{
	padding: 0px;
	margin: 0px;
	border:none
}</string>
  </property>
  <widget class="QLabel" name="lb_SumId">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>144</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
background-color: white;</string>
   </property>
   <property name="text">
    <string>Oner#KR222</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_Light">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1,
stop:0 rgba(255, 200, 200, 255), 
stop:1 rgba(255, 0, 0, 255));
border-radius: 10px;
border:none
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_Start">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>0</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_Pause">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="bt_Settings">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>20</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid lightgray;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_Pop">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>0</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FlowerView.qrc"/>
 </resources>
 <connections/>
</ui>
